src/app/public/modules/needs-attention/needs-attention.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="sky-elevation-1-bordered sky-padding-even-xl">
+  <h2 class="sky-font-heading-4">{{ 'Needs Attention' | skyLibResources }}</h2>
+  <sky-fluid-grid [gutterSize]="0">
+    <sky-row>
+      <sky-column *ngFor="let item of items; let isLast = last"
+        screenXSmall="12"
+        [screenSmall]="items.length > 6 ? 6 : 12"
+      >
+        <a
+          [href]="item.permalink.url"
+          [routerLink]="item.permalink.route"
+          class="
+            needs-attention-item
+            sky-margin-inline-lg
+            sky-padding-vertical-lg
+            sky-border-bottom-light
+          "
+        >
+          <sky-icon
+            icon="chevron-right"
+            class="needs-attention-item-icon"
+          ></sky-icon>
+          <span
+            class="needs-attention-item-label sky-padding-horizontal-sm"
```

Comment:
Have we checked the padding on the right side of the main label here if a message exists? To me it seems like there is a lot of padding there and it is a little odd that the underline is broken between the two. However, design knows those things better than me.